clang/lib/AST/DeclCXX.cpp

```diff
@@ -3423,6 +3425,13 @@ VarDecl *BindingDecl::getHoldingVar() const {
   return VD;
 }
 
+llvm::ArrayRef<Expr *> BindingDecl::getBindingPackExprs() const {
+  if (!Binding)
+    return {};
```

Comment:
Looking at how this is used, i think we can assert here instead

Reply:
Added. The assert was being triggered in one spot that I realized was unnecessary (when I tried removing the whole block of code that was going back and adding types to the DeclRefExprs.)